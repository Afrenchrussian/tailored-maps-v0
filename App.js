import React from 'react'
import { WebView } from 'react-native-webview'
import html from './build/index.html'

export default function App() {
	return (
		<WebView source={{ html: html }} />
	)
}
