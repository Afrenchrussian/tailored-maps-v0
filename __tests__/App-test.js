import React from 'react'
import renderer from 'react-test-renderer'
import App from '../App'

jest.mock('expo', () => ({
	AppLoading: 'AppLoading'
}))

describe('Main', () => {
	it('renders the loading screen', () => {
		expect(renderer.create(<App />).toJSON()).toMatchSnapshot()
	})

	it('renders the root without loading screen', () => {
		expect(renderer.create(<App skipLoadingScreen />).toJSON()).toMatchSnapshot()
	})
})
