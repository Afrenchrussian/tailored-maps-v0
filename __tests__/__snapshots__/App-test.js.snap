// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main renders the loading screen 1`] = `
<AppLoading
  onError={[Function]}
  onFinish={[Function]}
  startAsync={[Function]}
/>
`;

exports[`Main renders the root without loading screen 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
>
  <AppNavigator />
</View>
`;
